MobileCenter/MobileCenter/Internals/Sender/MSHttpSender.m

```diff
@@ -35,19 +35,22 @@ - (id)initWithBaseUrl:(NSString *)baseUrl
 
     // Construct the URL string with the query string.
     NSString *urlString = [baseUrl stringByAppendingString:apiPath];
-    NSURLComponents *components = [NSURLComponents componentsWithString:urlString];
-    NSMutableArray *queryItemArray = [NSMutableArray array];
+    __block NSString *queryStringForEncoding = @"";
```

Comment:
Maybe make `urlString` and `queryStringForEncoding` mutable strings to manipulate them more easily.